Python/Product/VSInterpreters/PackageManager/PipPackageManager.cs

```diff
@@ -110,6 +112,19 @@ int priority
             }
         }
 
+        private void EnsureActivated() {
+            if (_activatedEnvironmentVariables == null) {
+                var prefixPath = _factory.Configuration.GetPrefixPath();
+                if (_condaLocatorProvider != null && CondaUtils.IsCondaEnvironment(prefixPath)) {
+                    var rootConda = _condaLocatorProvider.FindLocator()?.CondaExecutablePath;
+                    var env = CondaUtils.CaptureActivationEnvironmentVariablesForPrefix(rootConda, prefixPath);
+                    _activatedEnvironmentVariables = env.Union(UnbufferedEnv).ToArray();
```

Comment:
I left it there because I liked keeping it consistent with `CondaPackageManager` and others that do something similar and do actually use it in more than on place. Plus it's very slightly better performance as a private static field (only create that dictionary once).